packages/migrate/src/__tests__/handlePanic.test.ts

```diff
@@ -13,6 +13,9 @@ import { Migrate } from '../Migrate'
 import { handlePanic } from '../utils/handlePanic'
 import CaptureStdout from './__helpers__/captureStdout'
 
+// follow this issue suggestion: https://github.com/aelbore/esbuild-jest/issues/26#issuecomment-893763840
```

Comment:
```suggestion
// hack to avoid the `TypeError: Cannot redefine property:`
// follow this issue suggestion: https://github.com/aelbore/esbuild-jest/issues/26#issuecomment-893763840
```

Reply:
Note: I have tried any combination of previous knowledge I had on Jest mocks, plus what I have read in the esbuild-jest issue mentioned there, but somehow I didn't get to mock sendPanic as I wanted too